modules/mux/targets/host/source/device.cpp

```diff
@@ -552,6 +554,12 @@ host::arch device_info_s::detectHostArch() {
 #elif defined(UTILS_SYSTEM_64_BIT)
   arch = host::arch::X86_64;
 #endif
+#elif defined(UTILS_SYSTEM_RISCV)
+#ifdef UTILS_SYSTEM_64_BIT
+  arch = host::arch::RISCV64;
+#elif defined(UTILS_SYSTEM_64_BIT)
+  arch = host::arch::RISCV32;
```

Comment:
This isn't right, this checks the wrong macro. Minor nit, we seem to consistently have 32-bit logic first, then 64-bit logic, but that's reversed here. Not a big deal but if changes are needed anyway, might as well do this too.